### PR TITLE
Added information about Content Design (on behalf of Allan O'Connor)

### PR DESCRIPTION
**Pull request by StephenHill-NHSBSA**
Added information about Content Design on behalf of Allan O'Connor.

Includes:

What is Content Design?
Why do it?
When to do it
Benefits
Example

### CODE REVIEW

File: src/design/index.md [hunk 1]
```diff
@@ -97,21 +103,75 @@ Interaction design along with other disciplines helps us create products that:
...
+* is not just about writing web copy
+* can take a range of formats
+* should be based on research
+
+### Why do it?
+
+Good content design:
+
+* helps people get the information they need in a way they expect
+* focuses on the users needs by creating content that is simple and easy to understand
```
Comment (yantantether) [inline src/design/index.md:143]: `user needs` or `users' needs` ?

File: src/design/index.md [hunk 2]
```diff
@@ -97,21 +103,75 @@ Interaction design along with other disciplines helps us create products that:
...
+* makes sure content is accessible and inclusive for anyone who needs it
+* keeps content across multiple services consistent by sticking using style guides and standards
```
Comment (yantantether) [inline src/design/index.md:145]: `sticking to style guides...`?

File: src/design/index.md [hunk 3]
```diff
@@ -97,21 +103,75 @@ Interaction design along with other disciplines helps us create products that:
...
+* gives our users the best possible experience of our services and reduces workloads for areas of the business
+
+### When to do it
+
+Content Design should work alongside the other areas of the UCD community from the Alpha stage through to the live environment to ensure the continuous improvement our services.
```
Comment (yantantether) [inline src/design/index.md:150]: Shouldn't this be `Alpha stage through to Live, to ensure...`
Alpha is a stage, and shouldn't compare to an environment. Environments are just part of the build pipeline

File: src/design/index.md [hunk 4]
```diff
@@ -97,21 +103,75 @@ Interaction design along with other disciplines helps us create products that:
...
+So that the user: 
+
+* is able to access information quickly
+* understands the subject matter 
+* isn’t excluded
+* gets the outcome they need 
+
+### Example
+
+A good example of how Content Design can help can be found in the Manage your service (MYS) portal.
```
Comment (yantantether) [inline src/design/index.md:167]: Capitalise `Manage Your Service`?

File: src/design/index.md [hunk 5]
```diff
@@ -97,21 +103,75 @@ Interaction design along with other disciplines helps us create products that:
...
+
+**Original content**:
+
+How many eligible people aged 65 and over, including those becoming age 65 by 31st March 2021, has the pharmacy vaccinated between 1st September 2020 and 31st January 2021?
```
Comment (yantantether) [inline src/design/index.md:171]: Can this and the next quote be block-quoted?

> Just use a greater than sign

File: src/design/index.md (hunk 1 above)
Comment (StephenHill-NHSBSA) [reply src/design/index.md:143]: image

File: src/design/index.md (hunk 2 above)
Comment (StephenHill-NHSBSA) [reply src/design/index.md]: image

File: src/design/index.md (hunk 4 above)
Comment (StephenHill-NHSBSA) [reply src/design/index.md]: image

**Comment by StephenHill-NHSBSA** [comment]
Resolved comments

File: src/design/index.md (hunk 3 above)
Comment (StephenHill-NHSBSA) [reply src/design/index.md]: image

**Review by yantantether — APPROVED** [review]

**Pull request merged**